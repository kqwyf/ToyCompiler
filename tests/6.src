/*abc*/
/*abc 123 */
/*      */
/* This is a multi-line comment.
*/
/* Another multi-line comment.
A comment.*/

/* Many comments.  /*
   Comments  */

/*    An multil-line endless comment.
      Endless.
