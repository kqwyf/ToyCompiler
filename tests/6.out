Line 14, Col 1: Incomplete comment. Enclosed automatically.

Token sequence:
/* ... */       < COMMENT     ,        >
/* ... */       < COMMENT     ,        >
/* ... */       < COMMENT     ,        >
/* ... */       < COMMENT     ,        >
/* ... */       < COMMENT     ,        >
/* ... */       < COMMENT     ,        >
/* ... */       < COMMENT     ,        >

Symbol table:
